utilities/rpp-performancetests/HIP_NEW/Tensor_hip_pkd3.cpp

```diff
@@ -1,25 +1,3 @@
-/*
-Copyright (c) 2019 - 2023 Advanced Micro Devices, Inc. All rights reserved.
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in
-all copies or substantial portions of the Software.
-
```

Comment:
Please remove changes to any files under the rpp-unittests or rpp-performance tests folders. These will not be updated henceforth. Just the test_suite folder must be updated to add relevant QA/unit/perf tests.